config/cms.php

```diff
@@ -93,6 +93,19 @@
 
     'backendSkin' => 'Backend\Skins\Standard',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Determines if logging in backend should run UpdateManager
```

Comment:
@Samuell1 Might be better to say "Automatically check for plugin updates on login".

Reply:
@bennothommo It doesnt automatically check for plugin updates, its just installs latest migrations that were not executed before.

Reply:
@Samuell1 Fair enough. What about "Automatically run migrations on login"?

I would prefer if it said what UpdateManager does as opposed to just saying "run UpdateManager".